generators/server/templates/src/main/java/package/config/oauth2/_OAuth2JwtAccessTokenConverter.java

```diff
@@ -0,0 +1,98 @@
+<%#
+ Copyright 2013-2017 the original author or authors.
```

Comment:
Can you update the copyright, have a look at the latest files, we've added "from the JHipster project" here. The issue is that we have people copy/pasting this, in some other projects, without respecting the license, and so in their Git history they are the "original author". And as you see we are taking care of everybody's work...